common/flatpak-context.c

```diff
@@ -1417,6 +1430,25 @@ option_unset_env_cb (const gchar *option_name,
   return TRUE;
 }
 
+static gboolean
+option_command_cb (const gchar *option_name,
+                   const gchar *value,
+                   gpointer     data,
+                   GError     **error)
+{
+  FlatpakContext *context = data;
+
+  if (value[0] == '\0')
+    {
+      g_set_error (error, G_OPTION_ERROR, G_OPTION_ERROR_FAILED,
+                   "Command cannot be empty");
+      return FALSE;
+    }
+
+  flatpak_context_set_command (context, value);
+  return TRUE;
```

Comment:
I think this should parse the `value` with `g_shell_parse_argv()` so that syntactically invalid commands can be rejected with a useful error message, even if the parsed value gets thrown away because it's stored as a single string internally.